rotkehlchen/externalapis/monerium.py

```diff
@@ -29,6 +34,13 @@
 # reprocessing any events that we already have.
 MAX_INDIVIDUAL_TX_QUERIES: Final = 4
 
+# TODO move this to env
+MONERIUM_API_BASE_URL: Final = 'https://api.monerium.app/'
+MONERIUM_ACCEPT_HEADER: Final = 'application/vnd.monerium.api-v2+json'
+MONERIUM_OAUTH_STORAGE_KEY: Final = 'monerium_oauth_credentials'
+TOKEN_REFRESH_MARGIN_SECONDS: Final = 60
+AUTHENTICATION_FLOW_CLIENT_ID: Final = '0cba773d-a369-11f0-b3a4-326caa0c532e'  # TODO: Change before going into production  # noqa: E501
```

Comment:
this is the only thing that should be moved to an env i guess?

Reply:
this will be hardcoded in the app. It is meant to be used for the authentication flow and can be here